.github/workflows/ci.yml

```diff
@@ -32,6 +32,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: shivammathur/setup-php@v2
         with:
+          ini-file: development
```

Comment:
This enables reporting of deprecation errors. In general the development ini is stricter than the production ini, so I think it's good/safe to use in CI.